src/Modules/OrchardCore.Commerce/Views/StripeApiSettings.Edit.cshtml

```diff
@@ -0,0 +1,21 @@
+@model StripeApiSettingsViewModel
+
+<p class="alert alert-warning">@T["The current tenant will be reloaded when the settings are saved."]</p>
+
+<div class="mb-3" asp-validation-class-for="PublishableKey">
+    <label asp-for="PublishableKey">@T["Publishable Key"]</label>
+    <input asp-for="PublishableKey" class="form-control" type="text" />
+    <span asp-validation-for="PublishableKey"></span>
+    <span class="hint">@T["Your Stripe's Publishable API key."]</span>
+</div>
+
+<div class="mb-3" asp-validation-class-for="SecretKey">
+    <label asp-for="SecretKey">@T["Secret Key"]</label>
+    <input asp-for="SecretKey" class="form-control" type="text" autocomplete="off" />
+    <span asp-validation-for="SecretKey"></span>
+    <span class="hint">@T["Your Stripe Secret API key."]</span>
+</div>
+
+<div class="mb-3">
+    <span class="hint">@T["Get your Stripe API keys at"] <a href="https://stripe.com/docs/keys" target="_blank">https://stripe.com/docs/keys</a></span>
```

Comment:
Include the link in the localization string. Don't forget the period at the end of the sentence.

Reply:
I meant make it a single expression, again in case of localization. `IHtmlLocalizer` strings are allowed to contain raw HTML.
```cshtml
<span class="hint">@T["Get your Stripe API keys at <a href="https://stripe.com/docs/keys" target="_blank">https://stripe.com/docs/keys</a>."] </span>
```